packages/primitives/core/src/positioning/types.ts

```diff
@@ -0,0 +1,30 @@
+import { ConnectionPositionPair } from '@angular/cdk/overlay';
+
+export enum RdxSide {
```

Comment:
Naming like `RdxPositionSide` / `RdxPositioningSide` will provide more context for later use, imo

Reply:
Nice, will change it 💯